generators/utils.js

```diff
@@ -577,7 +577,7 @@ function decodeBase64(string, encoding = 'utf-8') {
 }
 
 function loadYoRc(filePath = '.yo-rc.json') {
-    if (!jhiCore.FileUtils.doesFileExist(filePath)) {
+    if (!FileUtils.doesFileExist(filePath)) {
```

Comment:
Use `fs.existsSync` instead.

Reply:
There's a notable difference between the FileUtils' version and the Node FS' one: the latter only checks the path (https://nodejs.org/api/fs.html#fs_fs_existssync_path) and doesn't core if it's a folder or a regular file.

Reply:
I don't think we need that, I would prefer fs.existsSync for simplicity.
This kind of test is not done at yeoman-generator for `.yo-rc.json`, it will just throw if it cannot parse.

Anyway your call.

Reply:
In fact I think loadYoRc should throw if `.yo-rc.json` is a folder.
This is not a supported scenario at all, it's a bad problem and should not be ignored by returning undefined.